tests/test_io.py

```diff
@@ -33,7 +38,41 @@ def test_triplegs_from_to_csv(self):
             columns=['user_id', 'started_at', 'finished_at', 'geom'])
         assert filecmp.cmp(orig_file, tmp_file, shallow=False)
         os.remove(tmp_file)
-        
+
+    def test_Douglas_Peucker_Algorithm_reduces_triplet_length(self):
+        def plot_line(ax, ob):
+            x, y = ob.xy
+            ax.plot(x, y, alpha=0.7, linewidth=3, solid_capstyle='round', zorder=2)
+
+        pd.set_option('display.max_columns', 10)
+        pd.set_option('display.max_rows', 10)
+        orig_file = 'tests/data/triplegs_with_too_many_points_test.csv'
+        tpls = ti.read_triplegs_csv(orig_file, sep=';')
+        tpls_smoothed = smoothen_triplegs(tpls, epsilon=0.0001)
```

Comment:
We should add the smoothen_triplegs function as method for the as_triplegs accessor / class.